lib/request.js

```diff
@@ -563,13 +578,20 @@ AWS.Request = inherit({
    * @!macro nobrowser
    */
   createReadStream: function createReadStream() {
+    this.preventMultipleCalls();
```

Comment:
I might actually remove this line. This will cause an error to be thrown if createReadStream is called after a request is sent, but it would need to be wrapped in a try/catch. That might be confusing if the user expects to listen for an error on the stream instead.